CHANGELOG.md

```diff
@@ -4,20 +4,30 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+- No changes yet.
+
```

Comment:
This shouldn't be part of the changelog for this release. This should be in the Back to Development commit after this release.